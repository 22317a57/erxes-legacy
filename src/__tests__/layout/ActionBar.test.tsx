import { mount, shallow } from 'enzyme';
import * as React from 'react';
import * as renderer from 'react-test-renderer';

import ActionBar from '../../modules/layout/components/ActionBar';

describe('ActionBar component', () => {
  const defaultProps = {
    background: 'black'
  };

  test('renders ActionBar successfully', () => {
    shallow(<ActionBar {...defaultProps} />);
  });

  test('renders successfully with default value', () => {
    const wrapper = mount(<ActionBar {...defaultProps} />);
    const props = wrapper.props();

    expect(props).toMatchObject(defaultProps);
  });

  test('snapshot matches', () => {
    const rendered = renderer.create(<ActionBar {...defaultProps} />).toJSON();

    expect(rendered).toMatchSnapshot();
  });
});
