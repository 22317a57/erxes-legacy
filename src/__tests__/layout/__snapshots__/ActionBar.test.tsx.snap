// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionBar component snapshot matches 1`] = `
<div
  className="sc-htoDjs jjFcsU"
/>
`;
